en/test_and_ci/docker.md

```diff
@@ -126,13 +126,17 @@ xhost +
 # Run docker and open bash shell
 docker run -it --privileged \
 --env=LOCAL_USER_ID="$(id -u)" \
--v ~/src/PX4-Autopilot:/src/PX4-Autopilot/:rw \
+-v ~/Projects/PX4-Autopilot:/src/PX4-Autopilot/:rw \
 -v /tmp/.X11-unix:/tmp/.X11-unix:ro \
--e DISPLAY=:0 \
--p 14570:14570/udp \
---name=mycontainer px4io/px4-dev-ros:2017-10-23 bash
+-e DISPLAY=$DISPLAY \
```

Comment:
This would assume you are pulling the value of $DISPLAY from somewhere.  Why, and where from?

Reply:
From my limited testing on Ubuntu machines (20.04 desktop on a Laptop and 22.04 desktop on an Intel NUC) I found that `$DISPLAY` had already been set by something else within the system. The reason I chose to use this instead of `:0` was because on my laptop, the proper display to use was actually `:1` which ended up causing me a half hour or so of grief trying to figure it out. By using `$DISPLAY` I am assuming that other machines also have had this value automatically set and will hopefully mitigate potential issues for future users. I can go ahead and add a note mentioning this though so that if `$DISPLAY` is not set, users will at least have some understanding of what is going on (or I can change it back to `:0` and still include a note about possibly needing to change that value).

Reply:
I have proposed a new wording (with questions to be answered) here: https://github.com/PX4/PX4-user_guide/pull/2002/files#r947330948

Otherwise I think this PR is good to go. THanks!